daemon/daemon_linux.go

```diff
@@ -0,0 +1,336 @@
+package daemon
```

Comment:
I think this should called `daemon_unix.go` and include the build tag `+build !windows`.